src/Microsoft.Data.SqlClient.Extensions/Abstractions/test/SqlAuthenticationProviderTest.cs

```diff
@@ -0,0 +1,57 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Reflection;
+
+namespace Microsoft.Data.SqlClient.Extensions.Abstractions.Test;
+
+// Tests for the obsolete SqlAuthenticationProvider.GetProvider and SetProvider
```

Comment:
If you want to add comment blocks to test files, I won't complain, but please use xmldocs